infra/bin/infrastructure.ts

```diff
@@ -2,37 +2,15 @@
 import * as cdk from "aws-cdk-lib";
 import "source-map-support/register";
 import { EnvConfig } from "../lib/env-config";
-import { EnvType } from "../lib/env-type";
 import { FHIRServerStack } from "../lib/fhir-server-stack";
+import { initConfig } from "../lib/shared/config";
 
 const app = new cdk.App();
-//-------------------------------------------
-// Parse config based on specified env
-//-------------------------------------------
-async function getConfig(): Promise<EnvConfig> {
-  const env = app.node.tryGetContext("env");
-  const validVals = Object.values(EnvType);
-  if (!env || !validVals.includes(env)) {
-    throw new Error(
-      `Context variable missing on CDK command. Pass in as "-c env=XXX". Valid values are: ${validVals}`
-    );
-  }
-  const configPath = `../config/${env}.ts`;
-  const config = await import(configPath);
-  if (!config || !config.default) {
-    throw new Error(
-      `Ensure config is defined, could not fine file ${configPath}`
-    );
-  }
-  return config.default;
-}
```

Comment:
NDB, but my 2 cents would be to avoid making refactors like this

now, the cdk code will be structured differently in our other repos, and we'll have to make the same refactor in the other repos so it's consistent

Reply:
Did the same for OSS on [this PR](https://github.com/metriport/metriport/pull/372) - needed access to the config on more than one level down the stack and found easier to access w/ `getConfig()` than passing stuff around.

I can add this to internal as well if we keep this.